.pylintdict

```diff
@@ -270,6 +270,7 @@ moc
 mohij
 mohijkl
 mol
+møller
```

Comment:
I wonder if this special character might have a different ordering w.r.t. "o".

Reply:
Good point. So, I'm going to go with the ordering here https://en.wikipedia.org/wiki/Danish_and_Norwegian_alphabet. I did do some googling around to see if there's an agreed-upon ordering for all characters across all languages (to future proof the list) but didn't have much luck.

Reply:
To start with, and I think we have done it on occasions as well, is that this run by `sort` and I think that obeys, or looks at, what is defined for collating across multiple character sets. So while we more or less maintain this manually I might expect it to get run by sort in the future; or even regenerated since if we delete this, and assume all spellings are correct, then any thing picked will not be the in the standard dict and needs to be in such a file. I think we have done this in the past during refactor/moves etc. So it seems ok for now, and is possibly nothing to stress over too much.